### DISCUSSION TITLE
Floating labels

### DISCUSSION THREAD

**Discussion by martinbarilik** [orchidjs/tom-select#249] [Ideas]
Any plans to implement floating labels for default select size? 

https://getbootstrap.com/docs/5.1/forms/floating-labels/

**Comment by edgaryahir-angeles** [comment]
Hello, I leave you an example of how I did it, I hope it helps you.

https://jsfiddle.net/satdae/p9n40xmb/11/

**Comment by martinbarilik** [comment]
Hi, if i am gonna implement floating form into my project, i need it to be native in tom-select as i use hundreds of selects and inputs in view. 

But thanks for effort. 

Btw. styles are a bit off in your solution.

**Comment by mvxproject** [comment]
Related to : https://github.com/orchidjs/tom-select/discussions/337
